### PR TITLE
mikeio 0.6.0 release

### PR DESCRIPTION
**Pull request by georgebv**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/mikeio/meta.yaml
Comment (synapticarbors) [inline recipes/mikeio/meta.yaml:46]: It looks like the license file has been recently been added to the upstream `MANIFEST.in`, but it is not distributed in the current release. Please manually package the license file for now, and then when the next releases comes out you can remove it when you update the feedstock.

https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/mikeio/meta.yaml
Comment (georgebv) [reply recipes/mikeio/meta.yaml:46]: @synapticarbors thank you, this is a good idea. Now we need to resolve tests.

Comment (synapticarbors) [reply recipes/mikeio/meta.yaml:46]: It looks like the test data is not included in the source distribution. This would need to be fixed upstream. That said, conda-forge, doesn't require that you run the tests. The imports showing existence are usually enough. If you want to remove running the test suite for now, I can merge and then once the files are in-place in a future release, the feedstock can be updated to then run the tests again.

**Comment by georgebv** [comment]
@synapticarbors this would be great - I see where the issue is, tests fixtures will certainly need to be rewritten. If this is OK, please merge - I appreciate your help.

**Comment by synapticarbors** [comment]
@georgebv -- I just deleted the non-import tests. I'm assuming this should pass now. I'll also need the listed co-maintainer to approve being listed. If you have offline approval just let me know. Otherwise, they should comment on this PR.

**Comment by ecomodeller** [comment]
Please go ahead!

**Review by synapticarbors — APPROVED** [review]

**Pull request merged**
